src/gdriveApi.js

```diff
@@ -197,3 +198,35 @@ async function shareItem(id, shareData) {
 
   logger.log('verbose', 'gdrive - shareItem - done', id, shareData)
 }
+
+export async function getForeignCurrencyRates() {
+  const monthNames = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+  logger.log('verbose', 'gdrive - read spreadsheet')
```

Comment:
can be written as 
```suggestion
  logger.verbose('gdrive - read spreadsheet')
```
but it's a matter of preference

Reply:
I was just keeping the same style over the file